Readme.md

```diff
@@ -226,18 +226,29 @@ staging and dev environment.
 ### Resource
 
 For resource set the `partition` to `hca`, set your `service` name to the name of your component, and set the 
-`account-id` to the deployment stage. All other fields can be used as needed or use \* for wild cards. Resource names
- are case sensitive.
-
+`account-id` to the deployment stage. The field **resourcetype** can optionally be the name of a resource type defined
+ in fusillade. If `resourcetype` matches a resource type defined in fusillade then a resource policy will be used 
+ evaluating the users access. All other fields can be used as needed or use \* for wild cards. Resource names are 
```

Comment:
s/will be used evaluating the user access/will be used to evaluate user access/

Reply:
will fix